src/index.html

```diff
@@ -2,16 +2,196 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta  name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+      name="form"
+    >
+      <fieldset class="box_style">
+        <legend>Personal information</legend>
+
+        <div class="field_wrapper">
+          <label for="surname">Surname: </label>
+
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            value=" "
```

Comment:
Empty space as a default value in input is not a good practice. Please remove this attribute `value=" "` it's redundant here